### PR TITLE
Update README.md compared to recent changes in Iris

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
Do not merge this until after Iris has been released fully.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,17 @@
 
 ## How can I use this?
 
-Iris is alpha quality software and is highly incomplete. If you are not a developer or familiar with compiling and running Minecraft mods from source, OptiFine is likely the better choice in the immediate moment.
+**You can download Iris from [GitHub Releases](https://github.com/IrisShaders/Iris/releases), [Modrinth](https://modrinth.com/mod/iris) or [Curseforge]().**
```
Comment (DragonEggBedrockBreaking) [inline README.md:9]: I couldn't find the URL for the Iris curseforge page, so if someone could link it below that would be nice...

Comment (TheGamerX20) [reply README.md:9]: https://www.curseforge.com/minecraft/mc-mods/irisshaders
It should be this I think.

Comment (DragonEggBedrockBreaking) [reply README.md]: Thanks

File: README.md [hunk 2]
```diff
@@ -6,18 +6,17 @@
 
 ## How can I use this?
 
-Iris is alpha quality software and is highly incomplete. If you are not a developer or familiar with compiling and running Minecraft mods from source, OptiFine is likely the better choice in the immediate moment.
+**You can download Iris from [GitHub Releases](https://github.com/IrisShaders/Iris/releases), [Modrinth](https://modrinth.com/mod/iris) or [Curseforge](https://www.curseforge.com/minecraft/mc-mods/irisshaders).**
```
Comment (wafflecoffee) [inline README.md:9]: You forgot the oxford comma. Also, capitalise the F in CurseForge.

**Pull request closed without merging**